wft4galaxy/app/wizard.py

```diff
@@ -28,10 +28,13 @@
 _TEMPLATE_DIR = _os.path.join(_os.path.dirname(_os.path.abspath(__file__)), _os.pardir, _os.pardir, "templates")
 
 # configure module logger
-LogFormat = '%(asctime)s %(levelname)s: %(message)s'
-_logging.basicConfig(format=LogFormat)
-_logger = _logging.getLogger("WorkflowTest")
-_logger.setLevel(_logging.INFO)
+_logger = _common.default_logger
+
+# fix an issue related to the output buffering
+# when the wizard is running within a Docker container
+if _sys.version_info[0] == 2:
+    _logger.debug("Disabling output buffering...")
+    _sys.stdout = _os.fdopen(_sys.stdout.fileno(), 'w', 0)
```

Comment:
Consider wrapping this within a function that only gets called when the operation is relevant, for instance from `main()`.  There are cases where you wouldn't want a module to overwrite `sys.stdout` upon loading, such as when running unit tests.